lib/constants.js

```diff
@@ -112,6 +112,11 @@ function unquoted(text) {
     return text == null ? '' : text.replace (/(^")|("$)/g, '')
 }
 
+function quote(name) {
```

Comment:
Maybe the name should be more descriptive of what the method does - `quoteIfNeeded` for example.